data/persons.txt

```diff
@@ -768,3 +768,14 @@ ship "Ursa Polaris"
 	explode "big explosion" 40
 	explode "huge explosion" 50
 
+
+person "Subsidurial"
+	government "Indigenous Lifeform"
```

Comment:
I still don't believe that this should be part of the indigenous lifeform government, as killing this person ship will anger the Drak. Either this needs a separate government, or it needs 0 crew.

Reply:
OK. Crew reduced to 0.